packages/grid/src/vaadin-grid-styling-mixin.js

```diff
@@ -57,6 +86,18 @@ export const StylingMixin = (superClass) =>
       });
     }
 
+    /**
+     * Runs the `celPartNameGenerator` for the visible cells.
+     * If the generator depends on varying conditions, you need to
+     * call this function manually in order to update the styles when
+     * the conditions change.
+     */
+    generateCellPartNames() {
+      Array.from(this.$.items.children)
```

Comment:
This could now also use `iterateChildren`

Reply:
Done.